docs/app/utils.js

```diff
@@ -27,6 +35,7 @@ export const semanticUICSSRepoURL = 'https://github.com/Semantic-Org/Semantic-UI
 
 export const scrollToAnchor = () => {
   const anchor = location.hash && document.querySelector(location.hash)
+  const offsetY = window.scrollY || window.pageYOffset
```

Comment:
`pageYOffset` is alias for `scrollY`, but it's [supported](https://developer.mozilla.org/ru/docs/Web/API/Window/pageYOffset) from IE 9